worker/worker.go

```diff
@@ -277,12 +277,36 @@ const (
 //	           identifies group of workflow and activity implementations that are
 //	           hosted by a single worker process
 //	options  - configure any worker specific options like logger, metrics, identity
+//
+// DEPRCATED: use NewV2 instead since this implementation will panic on error
```

Comment:
typo and also change casing to `// Deprecated` as @Groxx mentioned in other comment

Reply:
fixed

Reply:
fixed